imap_data_access/cli.py

```diff
@@ -58,7 +58,6 @@ def _query_parser(args: argparse.Namespace):
     }
     output = imap_data_access.query(**query_params)
     num_files = len(output)
-    print(type(output))
     print(f"Found [{num_files}] files matching {query_params}:")
     print(output)
```

Comment:
Is the first line with what we found even useful, or should we just return the results so it can be piped somewhere else?